tests/test_traefik.py

```diff
@@ -124,3 +124,33 @@ def test_manual_ssl_config(tljh_dir):
             "whiteList": {"sourceRange": ["127.0.0.1"]}
         },
     }
+
+def test_extra_config(tmpdir, tljh_dir):
+    extra_config_dir = os.path.join(tljh_dir, config.CONFIG_DIR, "traefik_config.d")
+
+    state_dir = tmpdir.mkdir("state")
+
+    extra_config = {
```

Comment:
I would love for us to add a small integration test here as well - to make sure that traefik actually loads the extra config and applies it. For example, with `"dashboard": True`, we could maybe hit an endpoint to see if the dashboard is running? Or something like that... 

What do you think?

Reply:
Sure! This is a great idea :sparkles: 
Thanks Yuvi :heart: